src/EnergyPlus/SizingManager.cc

```diff
@@ -219,25 +219,6 @@ void ManageSizing(EnergyPlusData &state)
         state.dataGlobal->ZoneSizingCalc = true;
         Available = true;
 
-        if (state.dataSize->SizingFileColSep == CharComma) {
```

Comment:
Move to ZoneEquipmentManager to avoid creating an empty spsz output file. Can't make that decision here yet, because the project data (with Do Space Heat Balance flags) has not been read yet.

Reply:
:+1: